src/textual/app.py

```diff
@@ -864,19 +869,15 @@ async def _press_keys(self, keys: Iterable[str]) -> None:
                 await asyncio.sleep(float(wait_ms) / 1000)
             else:
                 if len(key) == 1 and not key.isalnum():
-                    key = (
-                        unicodedata.name(key)
-                        .lower()
-                        .replace("-", "_")
-                        .replace(" ", "_")
-                    )
+                    key = _character_to_key(key)
                 original_key = REPLACED_KEYS.get(key, key)
+                print(f"original key is {original_key}")
```

Comment:
Can we combine these two print statements? The intent is just to show a little information re the keys pressed in the docs build output.